src/unix/thread.c

```diff
@@ -72,10 +73,74 @@ int uv_thread_create(uv_thread_t *tid, void (*entry)(void *arg), void *arg) {
 }
 
 
+#if (defined(__APPLE__) && defined(__MACH__)) || defined(_AIX)
```

Comment:
This is ``(A && B) || C``. The parens around ``A && B`` are redundant, yes, but are there to add readability. Does this comment mean that you think them unnecessary?